main.go

```diff
@@ -33,11 +36,31 @@ func init() {
 	flag.StringVar(&address, "address", "localhost:8080", "Address of the integrations-registry service.")
 }
 
+type Config struct {
+	PackagesPath string `config:"packages.path"`
+	Foo          string `config:"foo"`
+}
+
 func main() {
 	flag.Parse()
 	log.Println("Integrations registry started.")
 	defer log.Println("Integrations registry stopped.")
 
+	cfg, err := ucfgYAML.NewConfigWithFile("config.yml")
```

Comment:
@urso For some reason this line stopped working for the `.` notations if I used `ucfg.PathSep(".")` as option. Did not investigate but somehow assume with the option is should work.